pkg/controller/operators/catalog/step.go

```diff
@@ -181,7 +187,7 @@ func (b *builder) NewCRDV1Beta1Step(client apiextensionsv1beta1client.Apiextensi
 				if k8serrors.IsNotFound(err) {
 					return v1alpha1.StepStatusNotPresent, nil
 				} else {
-					return v1alpha1.StepStatusNotPresent, errors.Wrapf(err, "error finding the %s CRD", crd.Name)
+					return v1alpha1.StepStatusNotPresent, fmt.Errorf("error finding the %q CRD: %w", crd.Name, err)
```

Comment:
minor nit, not blocking: the else statement here can be dropped as both branches are returning.

Reply:
If any non-nit changes come up, I can fix this while I'm at it. Otherwise, I'm inclined to leave this to avoid another CI cycle.

Relatedly, I'm pretty sure this kind of thing would be caught with go vet or golangci-lint. Perhaps in a follow-up, we should introduce one or both of those into our Makefile/CI process and get everything like this fixed at once.